src/engine/null/null_renderer.cpp

```diff
@@ -193,7 +193,7 @@ bool RE_BeginRegistration( glconfig_t *config, glconfig2_t *glconfig2 )
 	memset( config, 0, sizeof( glconfig_t ) );
```

Comment:
maybe it would be a good time to set this one too. The regular renderer uses `=` for glconfig too

Reply:
Ah, I didn't do it because it didn't trigger a warning, but it can't hurt.